SiteAssets/pages/3.0/scripts/configData.js

```diff
@@ -5,7 +5,6 @@ configData = {
     'Client Review': false, // Does the project require a client review to be included in the workflow?
     'Client Name': 'Client', // What is the name of the client of the project?
     'Archive hazards permissions': ['System admin'], // Which user roles should be authorised to archive hazards. This is given as a list. E.g., ['System admin', 'Designer', 'Construction Manager'].
-    'Sync Client Hazard Permissions': ['System admin'], // Defines who can use 'Sync Client Hazards' button.
```

Comment:
We need this line to be added back into the config file

Reply:
I've added it back